### DISCUSSION TITLE
Compatibility with V2-Analog

### DISCUSSION THREAD

**Discussion by bebrown** [hanshuebner/AppleII-VGA#17] [General]
Is the RPi Pico code from the V2 Analog project compatible with this one? They have a new modular firmware; will it work on this board?

Along the same lines, can I use the Atmel ATF16V8C chip in place of the PAL?

**Comment by mamejay** [comment]
You can use the same code for this this board.  Same for the PAL using a ATF16V8C chip.
I have had a few strange issues with the code and had to revert to slightly older firmware as it seemed that the some of my Pi Pico's did not like the firmware.  I have had these in a draw for over a year so not sure if they have some sort of older chips onboard.
